sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -991,6 +991,26 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val EXTENDED_EVENT_INFO = buildConf("spark.sql.extendedEventInfo")
```

Comment:
if possible, make those constant values in alphabeta order

Reply:
Changed the ordering of the added constants to be in alphabeta order with respect to one another.